enterprise/server/usage/BUILD

```
@@ -78,3 +78,36 @@ go_test(
         "@org_golang_x_sync//errgroup",
     ],
 )
+
+go_test(
+    name = "usage_test_postgres",
+    srcs = ["usage_test.go"],
+    args = [
+        "--testenv.database_type=postgresql",
+        "--testenv.reuse_server",
+    ],
+    exec_properties = {
+        "workload-isolation-type": "firecracker",
+        "init-dockerd": "true",
+        "recycle-runner": "true",
```

I realized that we could have only the test action run with docker-in-firecracker + runner recycling by applying a `test.` prefix to these. I made the changes in https://github.com/buildbuddy-io/buildbuddy/pull/4055 - mind applying this diff here as well?

```suggestion
        "test.workload-isolation-type": "firecracker",
        "test.init-dockerd": "true",
        "test.recycle-runner": "true",
```